lua/mini/misc.lua

```diff
@@ -465,4 +525,29 @@ H.is_number = function(x) return type(x) == 'number' end
 
 H.is_string = function(x) return type(x) == 'string' end
 
+H.cursor_jump = function()
```

Comment:
Possibly also not needed. See comment in `goto_last_cursor()`.

Reply:
Done.